api/src/services/users/users.ts

```diff
@@ -186,7 +186,7 @@ export const createUser: MutationResolvers['createUser'] = async ({
   return validateUniqueness(
     'user',
     { email },
-    { message: 'This email is already in use' },
+    { message: 'This email is already in use', db: db },
```

Comment:
Nice catch, this seems to be also mentioned in the [Redwood docs](https://docs.redwoodjs.com/docs/7.4/services#arguments)
"If you are overwriting the DATABASE_URL in your `src/lib/db` instantiation of the PrismaClient, you need to use this option. If not provided, a vanilla `new PrismaClient()` is used to run the callback that will not respect any custom configurations not represented in your `prisma.schema`"